src/pykeen/hpo/hpo.py

```diff
@@ -562,6 +565,10 @@ def hpo_pipeline(
         Keyword arguments to pass to the evaluator on instantiation
     :param evaluation_kwargs:
         Keyword arguments to pass to the evaluator's evaluate function on call
+    :param filter_validation_when_testing:
+        If true, during the evaluating of the test dataset, validation triples are added to the set of known positive
```

Comment:
```suggestion
        If true, during the evaluation on the test dataset, validation triples are added to the set of known positive
```

Reply:
Either `during evaluation on the test dataset` or `during the evaluation of the test dataset` would be correct. Which do you like better?

Reply:
Both fine :shrug: :smile:

Reply:
@cthoyt

Reply:
Done in f4596ddc